src/routes/v1/mod.rs

```diff
@@ -9,6 +11,23 @@ mod versions;
 pub fn v1_config(cfg: &mut web::ServiceConfig) {
     cfg.service(
         web::scope("api/v1")
+            .wrap_fn(|req, srv| {
+                let current_minute = Utc::now().timestamp() / 60;
```

Comment:
I'm pretty sure there's a .minutes method you can call directly

Reply:
Doesn't exist

Reply:
https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.minute

Reply:
Doesn't matter too much, if it works it works, this is temporary code anyway

Reply:
`Returns the minute number from 0 to 59.`

Reply:
```suggestion
                let current_minute = Utc::now().minute();
```